crates/core/component/ibc/src/component/state_key.rs

```diff
@@ -8,44 +8,17 @@ pub fn ibc_params() -> &'static str {
     "ibc/params"
 }
 
-// TODO (ava): move these to ibc-types eventually
+// these are internal helpers that are used by penumbra-ibc, but not part of the IBC spec (that is,
+// counterparties don't expect to verify proofs about them)
 pub fn client_processed_heights(client_id: &ClientId, height: &Height) -> String {
-    format!("ibc/clients/{client_id}/processedHeights/{height}")
+    format!("clients/{client_id}/processedHeights/{height}")
 }
 pub fn client_processed_times(client_id: &ClientId, height: &Height) -> String {
-    format!("ibc/clients/{client_id}/processedTimes/{height}")
+    format!("clients/{client_id}/processedTimes/{height}")
```

Comment:
if these aren't to be used by counterparties, and are component-internal, should we be prefixing them like the other component keys?